src/main/java/com/google/dicomwebfuse/dao/FuseDaoHelper.java

```diff
@@ -80,13 +83,22 @@
     }
 
     if (studyList.size() > MAX_STUDIES_IN_DICOM_STORE) {
-      throw new ToManyResultsException("Too large DICOM Store. In " + dicomPath.getDicomStoreId() +
+      LOGGER.warn("Too large DICOM Store. In " + dicomPath.getDicomStoreId() +
```

Comment:
Why are we making this change? It seems unrelated to the purpose of this CL, no?

Reply:
logic of throwing exception in case if folder have >15k objects breaks workflow because when we open /some/path/to/study OS checks if all folders from this path is exists, so we have to return code 0 for folder with >15k objects. we can't make /some/path/to/study available and /some/path unavailable in same time
it may be done how it works now - by showing downloaded result or we may show empty folder. not sure which way is less confusing for user.

Reply:
Makes sense. Could we just update the error message while we're here:

"DICOM Store %s has more than 15k studies in it, so only showing the first 15k"

Reply:
done